src/ripple/rpc/tests/KeyGeneration.test.cpp

```diff
@@ -164,10 +197,50 @@ class KeypairForSignature_test : public ripple::TestSuite
         testLegacySecret (secp256k1_strings.master_seed_hex);
     }
 
+    void testInvalidKeyType (char const* keyType)
+    {
+        testcase (keyType);
+
+        Json::Value params;
+        params[jss::key_type] = keyType;
+        params[jss::passphrase] = common::passphrase;
+
+        Json::Value error;
+        (void) keypairForSignature (params, error);
```

Comment:
It saves the reader from wondering why the function is called since it doesn't have any side effects.